pkg/support/dump.go

```diff
@@ -0,0 +1,128 @@
+package support
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/afero"
+	"k8s.io/utils/exec"
+)
+
+// AgentDumper is the interface for dumping runtime information of the agent. Its
+// functions should only work in an agent Pod or a Windows Node which has an agent
+// installed.
+type AgentDumper interface {
+	// DumpFlows should creates files that contains flows under the basedir.
```

Comment:
s/should creates/should create

here and everywhere else in the file

Reply:
Done.